src/Cookie.php

```diff
@@ -4,148 +4,150 @@
 
 final class Cookie
 {
-   private string $name;
+	private string $name;
 
-   private string $value;
+	private string $value;
 
-   private int $expires;
+	private int $expires;
 
-   private string $path;
+	private string $path;
 
-   private ?string $domain;
+	private ?string $domain;
 
-   private bool $secure;
+	private bool $secure;
 
-   private bool $httpOnly;
+	private bool $httpOnly;
 
-   private array $options;
+	private array $options;
 
-   public function __construct(string $name)
-   {
-        $this->name = $name;
-        $this->value = '';
-        $this->expires = 0;
-        $this->path = '/';
-        $this->domain = null;
-        $this->secure = false;
-        $this->httpOnly = true;
-        $this->options = [];
-    }
-
-    public static function fromString(string $name): self {
-        return new self($name);
-    }
+	public function __construct(
```

Comment:
It make sense to make the optional parameters while initialisation accessible as well.